api/composer.json

```diff
@@ -6,6 +6,7 @@
     "type": "project",
     "require": {
         "php": ">=7.1.3",
+        "codeception/aspect-mock": "*",
```

Comment:
@Zames-Chang 這邊有幾個問題我來確認一下

1. `composer.json` 中 require 的區塊是放 production code 需要的相關套件，如果是 aspect-mock 的話會擺放在 require-dev 中，你可以直接用 `composer require codeception/aspect-mock --dev` 來自動達到這件事情
2. 會使用 `*` 版本應該是他文件上標示這樣處理的，習慣上我們還是會限制住他的大版本，避免無意間升級的 breaking change
3. 我發現你 `composer.lock` 的 code change 數量蠻多的，有異動到一些不需要升級的套件，可以提供你 `composer` 的指令下法嗎？

下面我提供我異動的套件狀況

```
╭─hashman@hash-personal-ubuntu ~/repo/MOPCON/api ‹ruby-2.6.3› ‹feature/newsApi*›
╰─➤  composer require codeception/aspect-mock --dev
Using version ^3.0 for codeception/aspect-mock
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 7 installs, 0 updates, 0 removals
As there is no 'unzip' command installed zip files are being unpacked using the PHP zip extension.
This may cause invalid reports of corrupted archives. Besides, any UNIX permissions (e.g. executable) defined in the archives will be lost.
Installing 'unzip' may remediate them.
  - Installing nikic/php-parser (v4.2.4): Loading from cache
  - Installing goaop/parser-reflection (2.0.0): Loading from cache
  - Installing doctrine/cache (v1.8.0): Loading from cache
  - Installing doctrine/annotations (v1.7.0): Loading from cache
  - Installing jakubledl/dissect (v1.0.1): Loading from cache
  - Installing goaop/framework (2.3.2): Loading from cache
  - Installing codeception/aspect-mock (3.0.2): Loading from cache
```

謝謝

Reply:
我只下了 composer require codeception/aspect-mock:*